modules/twinklexfd.js

```diff
@@ -1437,7 +1660,12 @@ Twinkle.xfd.callbacks = {
 			targettalkpage.setCreateOption('recreate');
 			Twinkle.xfd.setWatchPref(targettalkpage, Twinkle.getPref('xfdWatchRelated'));
 			targettalkpage.setFollowRedirect(true);
-			targettalkpage.append();
+			// Add to userspace log even if not notifying the creator
+			if (params.lognomination && !params.usertalk) {
+				targettalkpage.append(Twinkle.xfd.callbacks.addToLog(params, null));
```

Comment:
Syntax error.

Reply:
Where?

Reply:
Lol, it should be `targettalkpage.append(function() { Twinkle.xfd.callbacks.addToLog(params, null)); });`  

Otherwise `addToLog` is evaluated immediately, not after the `append` has succeeded. No console error is raised as `addToLog` returns undefined, so morebits just thinks `append` did not get any argument.

Reply:
Haha oh geez, silly to miss that 🤕